zcash_test_vectors/orchard/commitments.py

```diff
@@ -25,14 +25,14 @@ def rcv_trapdoor(rand):
     return Scalar.random(rand)
 
 # https://zips.z.cash/protocol/nu5.pdf#concretesinsemillacommit
-def sinsemilla_commit(r: Scalar, D, M):
+def sinsemilla_commit(r: Scalar, D_hash, D_blind, M):
```

Comment:
Instead of overriding the existing function, we should add a new function for the new functionality.
Both functions are used in Orchard and both should be tested.